lib/Factory/DisplayFactory.php

```diff
@@ -205,6 +205,12 @@ public function query($sortOrder = null, $filterBy = [])
               SELECT display.displayId,
                   display.display,
                   display.defaultLayoutId,
+                  display.displayTypeId,
+                  display.screenSize,
+                  display.isOutdoor,
+                  display.customId,
```

Comment:
You will want to put these in `stringProperties` when you hydrate otherwise it will get converted to an int (we do that for all fields that end in `Id`